Gemfile.lock

```diff
@@ -214,7 +214,7 @@ GEM
       addressable (~> 2.3)
```

Comment:
Just to confirm, the command to unstage a file is **git reset <filepath>**? Would the filepath in this case be /Gemfile.lock?